### DISCUSSION TITLE
Disable automatic email sending in favor of custom integration?

### DISCUSSION THREAD

**Discussion by stefan-girlich** [nhost/nhost#2657] [Q&A] [answered]
My team and I started off a project using Supabase, but are currently considering switching to a different BaaS solution since (among other issues) it is not trivial to integrate an mailer service (e.g. Brevo) via REST (see [my comment here](https://github.com/supabase/auth-js/issues/290#issuecomment-2051649014)).
In a nutshell: When a user register (or requests a password change etc.) I do not want nhost to send an email based on an internal template. Instead, I want to trigger a function which passes the corresponding link to an external service via REST/HTTP which then sends the actual email.

Would you know if this is possible? How could this be implemented in nhost?

Thanks a lot for your help!

**Comment by dbarrosop** [comment, marked as answer]
Hello, two different problems:

> disable sending emails

Not currently supported. A quick workaround would be to run a local mailhog (or similar) with [Nhost Run](https://nhost.io/product/run) so the emails go nowhere. We could, of course, implement a setting to disable sending emails. The only question is feature priorization (hence why I mentioned the mailhog workaround).

> I want to trigger a function which passes the corresponding link to an external service via REST/HTTP which then sends the actual email

This is possible. You can create an event that is triggered based on some database columns being updated so you could just simply create an event for the relevant column (`auth.users.ticket`).

If you have more questions don't hesitate to let us know.
